src/bundle/Controller/LocationController.php

```diff
@@ -494,7 +494,7 @@ public function removeAction(Request $request): Response
 
                 foreach ($data->getLocations() as $locationId => $selected) {
                     $location = $this->locationService->loadLocation($locationId);
-                    $this->locationService->deleteLocation($location);
+                    $this->trashService->trash($location);
```

Comment:
In the trash, we have the restore button, how this will work if the parent is in the trash?

Reply:
It will allow restoring the child content object under a new parent location, ref:

<img width="1098" alt="Zrzut ekranu 2021-02-16 o 09 39 41" src="https://user-images.githubusercontent.com/34310128/108038252-e823b380-703a-11eb-8f40-da36e191cacd.png">